HLTrigger/Configuration/python/customizeHLTForPFTrackingPhaseI2017.py

```diff
@@ -0,0 +1,294 @@
+	        'BPix1+BPix2+FPix1_pos',
+	        'BPix1+BPix2+FPix1_neg',
+	        'BPix1+FPix1_pos+FPix2_pos',
+	        'BPix1+FPix1_neg+FPix2_neg' 
+	    ),
+	    MTOB = cms.PSet(  ),
+	    TEC = cms.PSet(  ),
+	    MTID = cms.PSet(  ),
+	    FPix = cms.PSet( 
+	      HitProducer = cms.string( "hltSiPixelRecHits" ),
+	      hitErrorRZ = cms.double( 0.0036 ),
+	      useErrorsFromParam = cms.bool( True ),
+	      TTRHBuilder = cms.string( "hltESPTTRHBuilderPixelOnly" ),
+	      skipClusters = cms.InputTag( "hltIter2ClustersRefRemoval" ),
+	      hitErrorRPhi = cms.double( 0.0051 )
+	    ),
+	    MTEC = cms.PSet(  ),
+	    MTIB = cms.PSet(  ),
+	    TID = cms.PSet(  ),
+	    TOB = cms.PSet(  ),
+	    BPix = cms.PSet( 
+	      HitProducer = cms.string( "hltSiPixelRecHits" ),
+	      hitErrorRZ = cms.double( 0.006 ),
+	      useErrorsFromParam = cms.bool( True ),
+	      TTRHBuilder = cms.string( "hltESPTTRHBuilderPixelOnly" ),
+	      skipClusters = cms.InputTag( "hltIter2ClustersRefRemoval" ),
+	      hitErrorRPhi = cms.double( 0.0027 )
+	    ),
+	    TIB = cms.PSet(  )
+	)
+	process.hltIter2PFlowPixelSeeds.OrderedHitsFactoryPSet = cms.PSet( 
+	  maxElement = cms.uint32( 0 ),
+	  ComponentName = cms.string( "StandardHitTripletGenerator" ),
+	  GeneratorPSet = cms.PSet( 
+	    useBending = cms.bool( True ),
+	    useFixedPreFiltering = cms.bool( False ),
+	    maxElement = cms.uint32( 100000 ),
+	    phiPreFiltering = cms.double( 0.3 ),
+	    extraHitRPhitolerance = cms.double( 0.032 ),
+	    useMultScattering = cms.bool( True ),
+	    ComponentName = cms.string( "PixelTripletHLTGenerator" ),
+	    extraHitRZtolerance = cms.double( 0.037 ),
+	    SeedComparitorPSet = cms.PSet(  ComponentName = cms.string( "none" ) )
+	  ),
+	  SeedingLayers = cms.InputTag( "hltIter2PixelLayerTriplets" )
+	)
+	process.hltIter2PFlowPixelSeeds.SeedCreatorPSet = cms.PSet(  refToPSet_ = cms.string( "HLTSeedFromConsecutiveHitsTripletOnlyCreator" ) )
+	
+	process.HLTIterativeTrackingIteration2 = cms.Sequence( process.hltIter2ClustersRefRemoval + process.hltIter2MaskedMeasurementTrackerEvent + process.hltIter2PixelLayerTriplets + process.hltIter2PFlowPixelSeeds + process.hltIter2PFlowCkfTrackCandidates + process.hltIter2PFlowCtfWithMaterialTracks + process.hltIter2PFlowTrackCutClassifier + process.hltIter2PFlowTrackSelectionHighPurity )
+
+
+	for seqName in process.sequences:
+		seq = getattr(process,seqName)
+		from FWCore.ParameterSet.SequenceTypes import ModuleNodeVisitor
+		l = list()
+         	v = ModuleNodeVisitor(l)
+          	seq.visit(v)		
+		if process.hltPixelTracks in l and not process.hltPixelLayerQuadruplets in l:
+			seq.remove(process.hltPixelLayerTriplets)
+			index = seq.index(process.hltPixelTracks)
+			seq.insert(index,process.hltPixelLayerQuadruplets)
+	
+	from RecoTracker.Configuration.customiseForQuadrupletsByCellularAutomaton import customiseForQuadrupletsByCellularAutomaton
+	process = customiseForQuadrupletsByCellularAutomaton(process)
```

Comment:
Can I also ask why to call another customize function instead of putting in the CA directly in here? Do you want to easily switch between "PixelQuadrupletGenerator" and CA? Or is there some other motivation?